Makefile

```
@@ -1,7 +1,13 @@
-TARGET := iphone:clang:latest:16.0
-
-THEOS_PACKAGE_SCHEME = rootless
-ARCHS = arm64
+ARCHS = arm64 arm64e
+
+ifeq ($(THEOS_PACKAGE_SCHEME), rootless)
+	TARGET := iphone:clang:latest:16.0
```

```suggestion
	TARGET := iphone:clang:latest:15.0
```
Since you recommended using 15 instead of 16, I'll try that out in the CI of this branch to see if it seems to compile fine.